depends/hosts/darwin.mk

```diff
@@ -0,0 +1,17 @@
+OSX_MIN_VERSION=10.7
+OSX_SDK_VERSION=10.9
+OSX_SDK=$(SDK_PATH)/MacOSX$(OSX_SDK_VERSION).sdk
+LD64_VERSION=241.9
```

Comment:
Thanks for clarifying this and the SDK comment. Much appreciated. Didn't know about cctools-port. Will have to take a closer look at that one.